### PR TITLE
Fix rabbit

### PR DESCRIPTION
**Pull request by stvstnfrd**
@caesar2164 These changes appear to have gone missing in the merge.

### CODE REVIEW

File: stanford/data-ansible.cfg [hunk 1]
```diff
@@ -5,6 +5,7 @@ host_key_checking = False
 hostfile=./ec2.py
 jinja2_extensions=jinja2.ext.do
 log_path=~/data-ansible.log
+roles_path = ../playbooks/roles:./roles
```
Comment (caseylitton) [inline stanford/data-ansible.cfg]: what's this syntax mean path:path

Comment (stvstnfrd) [reply stanford/data-ansible.cfg]: It's the same as the shell `$PATH` variable [1].

[1] https://unix.stackexchange.com/questions/26047/how-to-correctly-add-a-path-to-path

Comment (stvstnfrd) [reply stanford/data-ansible.cfg]: aka: colon-delimited list

File: stanford/roles/cluster/defaults/main.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+---
+CLUSTER_AMI: ami-73f7da13  # 16.04
+CLUSTER_AMI: ami-89f3dee9  # 14.04
```
Comment (caseylitton) [inline stanford/roles/cluster/defaults/main.yml]: Why two CLUSTER_AMI variables?

File: stanford/roles/cluster/defaults/main.yml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  - "{{ CLUSTER_SUBNET_ID }}"
+CLUSTER_ELBS:
+  - name: "{{ CLUSTER_DEPLOYMENT }}-elb-{{ CLUSTER_NAME }}"
+    subnets: "{{ CLUSTER_ELB_SUBNETS }}"
+    listeners: "{{ CLUSTER_ELB_LISTENERS }}"
+    health_check: "{{ CLUSTER_ELB_HEALTH_CHECK }}"
+    scheme: internal
+CLUSTER_INSTANCE_TYPE: t2.micro
+CLUSTER_KEY: "{{ CLUSTER_DEPLOYMENT }}-{{ CLUSTER_NAME }}"
+CLUSTER_NAME: 'TODO'
```
Comment (caseylitton) [inline stanford/roles/cluster/defaults/main.yml]: todo?

Comment (stvstnfrd) [reply stanford/roles/cluster/defaults/main.yml]: Just a placeholder.

File: stanford/roles/cluster/defaults/main.yml (hunk 2 above)
Comment (stvstnfrd) [reply stanford/roles/cluster/defaults/main.yml]: At the time, we were juggling between both 16.04 and 14.04 builds.
Now, we're 16.04 only.

I can remove the 2nd var now.

File: stanford/sandbox/ansible.cfg [hunk 4]
```diff
@@ -1,5 +1,6 @@
 [defaults]
 host_key_checking = False
+inventory = ../../playbooks/edx-east/ec2.py
```
Comment (caesar2164) [inline stanford/sandbox/ansible.cfg]: should the `edx-east` here be changed?

File: stanford/roles/cluster/defaults/main.yml [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+CLUSTER_ELB_HEALTH_CHECK:
+  ping_path: '/'
+  ping_protocol: http
+  ping_port: 80
+  response_timeout: 5  # seconds
+  interval: 30  # seconds
+  unhealthy_threshold: 2  # attempts
+  healthy_threshold: 10  # attempts
+CLUSTER_ELB_LISTENERS:
+  - protocol: http
```
Comment (caesar2164) [inline stanford/roles/cluster/defaults/main.yml]: not `https`?

File: stanford/roles/cluster/defaults/main.yml (hunk 2 above)
Comment (stvstnfrd) [reply stanford/roles/cluster/defaults/main.yml]: I removed the var and updated the AMI to a more recent 16.04

File: stanford/data-ansible.cfg (hunk 1 above)
Comment (stvstnfrd) [reply stanford/data-ansible.cfg]: Sadly, you can't use variables in an ansible config file.
Also, 3 of these files will be deleted with #110

File: stanford/sandbox/ansible.cfg (hunk 4 above)
Comment (stvstnfrd) [reply stanford/sandbox/ansible.cfg]: No, we're using edx's file.

File: stanford/roles/cluster/defaults/main.yml (hunk 5 above)
Comment (stvstnfrd) [reply stanford/roles/cluster/defaults/main.yml]: This is overridden elsewhere; it's just a placeholder.

**Pull request merged**